### PR TITLE
DOCSP-15552: update compatibility tables for motor v2.4

### PR DESCRIPTION
**Pull request by ccho-mongodb**
## Pull Request Info

Includes trailing whitespace removal.

### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-15552

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/06638ec/drivers/docsworker-xlarge/DOCSP-15552-motor-v2.4/motor/

### CODE REVIEW

**Review by kyuan-mongodb — COMMENTED** [review]
Looking at https://motor.readthedocs.io/en/2.4.0/requirements.html, it seems that only 2.4 should have Python 3.9 checked off.  You have 2.2-2.4 checked off here.

**Comment by ccho-mongodb** [comment]
> https://motor.readthedocs.io/en/2.4.0/requirements.html

Good catch! I wonder if that was an update that we missed or if they retroactively changed something.

File: source/includes/language-compatibility-table-motor.rst [hunk 1]
```diff
@@ -135,86 +91,59 @@
...
-
-   * - 0.5
+     - |checkmark|
      -
      -
+
+   * - 1.2
+     -
      -
      -
```
Comment (kyuan-mongodb) [inline source/includes/language-compatibility-table-motor.rst:101]: For 3.7, it should also have 1.2
```suggestion
     - |checkmark|
```

**Review by kyuan-mongodb — APPROVED** [review]
Just one box needs to be fixed, but besides that, LGTM

**Pull request merged**